Sentinel/Classes/Core/PreferencesInfo/OptionToggleView.swift

```diff
@@ -9,9 +9,10 @@ import SwiftUI
 
 struct OptionToggleView: View {
 
-    @State var value: Bool
+    @State private var value: Bool
     let title: String
     let onValueChanged: (Bool) -> Void
+    let getter: () -> Bool
```

Comment:
Which issue is fixed with adding a `getter` which is called on `onAppear`? Does `item.loadStoredValue()` not provide the correct value, and if not, in which cases?